app/models/rides.py

```diff
@@ -1,62 +1,222 @@
 class rides:
-	def __init__(self):
```

Comment:
Why did you delete the `__init__`???

Reply:
I decided to use the attributes declared in the initialise method in the add method instead. So there was no need of initialising them in the contractor